iter8/README.md

```diff
@@ -0,0 +1,56 @@
+# Iter8
+
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 automates traffic control for new versions of apps/ML models in the cluster.
+
+Iter8 comes with one component:
+
+1. [Iter8](#Iter8)
+
+## Iter8
+
+Contains deployment manifests for the Iter8 controller.
+
+- [iter8-controller](https://github.com/iter8-tools/iter8)
+
+## Iter8 architecture
+
+Iter8 has a controller that automatically reconfigures the Service Mesh when the set of versions of a model change. This capability enables blue-green and canary rollouts of new versions of a model.
+
+## Installation process
+Following are the steps to install Iter8 as a part of OpenDataHub install:
+
+1. Install the OpenDataHub operator.
+2. Create a KfDef that includes the Iter8 component.
+
+```yaml
+apiVersion: kfdef.apps.kubeflow.org/v1
```

Comment:
Taneem indicates that ODH v2.0 operator moved away from kfdef towards kustomize. Is this README still WIP? Will `kfdefs` eventually go away from this README to be replaced entirely by kustomize?

Reply:
yes, kfdef will go away so kustomize will be the main command to generate manifests.